src/main.cpp

```diff
@@ -18,6 +18,7 @@
 #include "init.h"
 #include "merkleblock.h"
 #include "net.h"
+#include "netbase.h"
```

Comment:
@rebroad An easy way to find out is to remove the line and compile. The compiler will tell you what is used but missing.